src/main/java/Utils/GetEnv.java

```diff
@@ -0,0 +1,19 @@
+package Utils;
+import javassist.NotFoundException;
+import java.util.Optional;
+
+public class GetEnv {
+
+    public static String getEnvLogin() throws NotFoundException {
+        String mvclogin;
+        return mvclogin = Optional.ofNullable(System.getenv("MVCLOGIN"))
```

Comment:
it's better to catch exceptions in GetEnt class,  so won't catch it in Hibernate Util